ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -498,7 +498,6 @@ export default function ConfirmationPage({
       <TemplateAlertContextProvider
         pendingConfirmation={pendingConfirmation}
         onSubmit={!templatedValues.hideSubmitButton && handleSubmit}
-        onCancel={templatedValues.onCancel}
```

Comment:
Is this because cancelling just closes the modal and returns to the original confirmation?

Reply:
Yep alert modal has only 2 buttons:
1. Submit
2. Review pending confirmations